application/src/main/java/io/vertx/core/application/VertxApplication.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2011-2023 Contributors to the Eclipse Foundation
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ */
+
+package io.vertx.core.application;
```

Comment:
we need to use a different package name, than `io.vertx.core`, e.g `io.vertx.launcher seems more appropriate